openfl/transport/grpc/director_client.py

```diff
@@ -191,31 +190,35 @@ def set_new_experiment(self, name, col_names, arch_path,
                            initial_tensor_dict=None):
         """Send the new experiment to director to launch."""
         logger.info('SetNewExperiment')
-        model_proto = None
         if initial_tensor_dict:
```

Comment:
What happens if the function receives no tensor dict? 
It probably never happens.

Reply:
Function return `None`.
It looks like that it is not part of current task.